src/abstract.jl

```diff
@@ -29,6 +29,9 @@ However, any concrete Lattice type should have at least the following fields:
 """
 abstract type Lattice end
 
+numSites(l::Lattice) = l.sites
```

Comment:
The camel casing is against the Julia style guide rules (and I personally don't like it much). If we need this getter method, we should better call it `num_sites` (or maybe even `length`?) and define `num_sites(lattice::LatPhysBase.AbstractLattice) = numSites(lattice)`.

Reply:
Another idea that comes to mind is adjusting our (abstract) definition of a lattice to make it compatible with the corresponding one in LatticePhysics. After all, we don't yet have many lattices here.

Reply:
Also, just to be safe, we should prefix with `@inline`.

Reply:
Names have been changed, `@inline` has been added as of d9e65ad